File: SIMINOV-HYBRID-PHONEGAP-TEMPLATE/assets/www/Siminov/Authorization/Credential.js


function Credential() {

	//Class Variables.
	var accountId = null;
	var token = null;
	var active;

	getAccountId = function() {
		return accountId;
	}

	setAccountId = function(val) {
		accountId = val;
	}

	getToken = function() {
		return token;
	}

	setToken = function(val) {
		token = val;
	}

	getActive = function() {
		return active;
	}
	
	isActive = function() {
		return active;
	}

	setActive = function(val) {
		active = val;
	}
}


Credential.ACCOUNT_ID = "ACCOUNT_ID";
Credential.TOKEN = "TOKEN";
Credential.ACTIVE = "ACTIVE";


FunctionUtils.extend(Database, Credential);

File: SIMINOV-CONNECT-TEMPLATE/src/siminov/connect/template/model/Credential.java
package siminov.connect.template.model;

import siminov.orm.database.Database;


public class Credential extends Database {

	//Column Names.
	public static final String ACCOUNT_ID = "ACCOUNT_ID";
	public static final String TOKEN = "TOKEN";
	public static final String ACTIVE = "ACTIVE";


	//Class Variables.
	private String accountId = null;
	private String token = null;
	private boolean active;

	public String getAccountId() {
		return this.accountId;
	}

	public void setAccountId(String accountId) {
		this.accountId = accountId;
	}

	public String getToken() {
		return this.token;
	}

	public void setToken(String token) {
		this.token = token;
	}

	public boolean getActive() {
		return this.active;
	}
	
	public boolean isActive() {
		return this.active;
	}

	public void setActive(boolean active) {
		this.active = active;
	}
}
